### PR TITLE
Add 3 icons

### PR DESCRIPTION
**Pull request by benteg**
# Description

Adds icons for Eternity, MRepo and Overload.

### Icons added
* Eternity (`eu.toldi.infinityforlemmy`)
* MRepo (`com.sanmer.mrepo`)
* Overload (`cloud.pablos.overload`)

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

File: svgs/overload.svg
Comment (x9136) [inline svgs/overload.svg:1]: It's worth increasing the thickness of the arrows to 10px to make them look more appropriate relative to the suitcase. I also suggest raising the icon by 1-2px to balance the void at the top.

**Review by x9136 — APPROVED** [review]
Thank you for contributing! If it's not too much trouble, please correct the minor things, although technically the PR can be accepted.

File: app/assets/appfilter.xml
Comment (x9136) [inline app/assets/appfilter.xml:1]: Screenshot 2023-10-14 at 14 04 57

Need a space.

**Review by x9136 — APPROVED** [review]

**Review by x9136 — APPROVED** [review]
I did.

**Comment by benteg** [comment]
I just finished remaking the eternity icon.
eternity

**Comment by x9136** [comment]
@benteg Nice one!

I assumed that you decided to leave the PR as is, so I finalized it myself. So I suggest you create a new PR to update the icon(s) if you want.

**Comment by benteg** [comment]
No problem. I was busy with school and only had time again today. New pr: #1534

**Pull request merged**